pilot/docker/Dockerfile.proxy_debug

```diff
@@ -1,34 +1,14 @@
-FROM ubuntu:xenial
-# Do not add more stuff to this list that isn't small or critically useful.
-# If you occasionally need something on the container do
-# sudo apt-get update && apt-get whichever
-RUN apt-get update && \
-    apt-get install --no-install-recommends -y \
-      curl \
-      iptables \
-      iproute2 \
-      iputils-ping \
-      dnsutils \
-      netcat \
-      tcpdump \
-      net-tools \
-      libc6-dbg gdb \
-      elvis-tiny \
-      lsof \
-      busybox \
-      sudo && \
-    rm -rf /var/lib/apt/lists/*
+FROM istionightly/base_debug
```

Comment:
Shouldn't this be a tagged image for reproducibility? How are we going to rebuild this image in a month time?

Reply:
We can tag it - in a separate PR. This is not a regression - before an apt-get update was also made,
I just moved that to a different image.

This doesn't have to be pushed or downloaded - part of the build you can rebuild.